Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,12 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def determine_dynamic_linker_path(formula)
+    if formula && ["glibc", "glibc25"].include?(formula.name)
```

Comment:
Updates. However, I have a wild idea. Would it be better if we have portable version of binutils and even glibc which can be poured everywhere on the file system?